_posts/2020-09-15-open-sourcing-a-private-project.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "How to Open Source a Closed Source Project"
+date: 2020-09-17 11:00:00
+categories: ["open-source"]
+author: "fionaDL"
+---
+
+Open Source is one of the things that we [value]( https://www.ombulabs.com/blog/values/our-values.html) as a company. Our philosophy is that “everything we do should be open by default”. This is why in the past few months we decided to open source two of our internal projects. We built [Dash](https://github.com/fastruby/dash) and [Points](https://github.com/fastruby/points) to make our daily processes smoother, and then we thought “hey, why not share them with everyone?”.
+Now we also want to share the process of how we turned them from private to open source.
+
+<!--more-->
+
+### Making a list
```

Comment:
@FionaDL Could you make sure you use h2 instead of h3? In terms of readability and SEO, it makes more sense to follow always the same hierarchy in our articles. The title is h1. 

```suggestion
## Making a list
```